mesh/pmesh.hpp

```diff
@@ -211,9 +208,17 @@ class ParMesh : public Mesh
    /// Internal function used in ParMesh::MakeRefined (and related constructor)
    void MakeRefined_(ParMesh &orig_mesh, int ref_factor, int ref_type);
 
+   /// Swaps internal data with another ParMesh, including non-geometry members.
+   /// See @a Mesh::Swap
+   void Swap(ParMesh &other);
+
    void Destroy();
 
 public:
+   /// Default constructor. Create an empty @a ParMesh.
+   ParMesh() : MyComm(0), NRanks(0), MyRank(-1),
+      have_face_nbr_data(false), pncmesh(NULL) {}
```

Comment:
I just noticed that `glob_elem_offset` and `glob_offset_sequence` were not added to the initialization list of this constructor when they were added in another PR. We can fix that here.

Reply:
These can just be initialized to zero, right?

Reply:
In the other constructors they seem to be initialized to `-1`, so we probably should do the same here.

Reply:
Done